tests/row/test_query_builder.py

```diff
@@ -0,0 +1,22 @@
+from pyfreeleh.row.base import Ordering
+from pyfreeleh.row.query_builder import GoogleSheetQueryBuilder
+
+
+def test_query_builder() -> None:
+    query = GoogleSheetQueryBuilder().build_select(["B", "C"])
+    assert query == "SELECT B,C WHERE A IS NOT NULL"
+
+    query = GoogleSheetQueryBuilder().limit(10).build_select(["B"])
+    assert query == "SELECT B WHERE A IS NOT NULL LIMIT 10"
+
+    query = GoogleSheetQueryBuilder().offset(10).build_select(["B"])
+    assert query == "SELECT B WHERE A IS NOT NULL OFFSET 10"
+
+    query = GoogleSheetQueryBuilder().order_by(Ordering.ASC("A"), Ordering.DESC("B")).build_select(["A", "B"])
+    assert query == "SELECT A,B WHERE A IS NOT NULL ORDER BY A ASC, B DESC"
```

Comment:
I think can consider to reverse the `A` and `B` order by clause so that the test covers the logic of preserving the ordering.

Reply:
> preserving the ordering.

Are you expecting the generated query of `.build_select(["B", "A"])` to be `SELECT A,B` or `SELECT B,A`?

Reply:
When iIpass `build_select(['B','A'])` what did you expect the actual query going to be?

`SELECT A,B` or `SELECT B,A`

I believe the ordering here doesn't matter as the GViz API will return it as a map.

Reply:
I'm referring to the "order by" clause, not the columns to be selected. The columns to be selected has no ordering. But the order by clause has ordering.

Reply:
Done.